src/pages/{mdx.frontmatter__permalink}.js

```diff
@@ -1,24 +1,43 @@
-import * as React from 'react'
-import { MDXRenderer } from 'gatsby-plugin-mdx'
-import { graphql, Link } from 'gatsby'
-import AuthorsYAMLData from "../../content/authors.yaml"
-import CategoriesYAMLDATA from "../../content/categories.yaml"
+import * as React from 'react';
+
+import { MDXRenderer } from 'gatsby-plugin-mdx';
+import { graphql, Link } from 'gatsby';
+
+import AuthorsYAMLData from "../../content/authors.yaml";
+import CategoriesYAMLDATA from "../../content/categories.yaml";
+import { Category } from "../components/category";
+
+import * as styles from './blog-post.module.scss';
+
 const _ = require("lodash")
 
 const BlogPost = ({ data }) => {
-  const author = AuthorsYAMLData.find(author => { return author.author === data.mdx.frontmatter.author })
-  const category = CategoriesYAMLDATA.find(category => { return category.category === data.mdx.frontmatter.category })
+  const author = AuthorsYAMLData.find(author => { return author.author === data.mdx.frontmatter.author });
+  const category = CategoriesYAMLDATA.find(category => { return category.category === data.mdx.frontmatter.category });
+
   return (
-    <>
-        <div><Link to={`/categories/${ _.kebabCase(category.category) }`}>{ category.title }</Link></div>
-        <div><Link to={`/authors/${ _.kebabCase(author.author) }`}>{ author.display_name }</Link></div>
-        <div>{ author.image }</div>
-        <div>{ author.profile_url }</div>
-        <p>{data.mdx.frontmatter.date}</p>
-        <MDXRenderer>
-            {data.mdx.body}
-        </MDXRenderer>
-    </>
+    <div className={styles.container}>
+      <div className={styles.headerContainer}>
+        <Category data={category.display_name}/>
```

Comment:
Why do you use snake case?

Reply:
Actually, there is no reason, they wrote the code like this. What I can do is to create a function that changes the type of writing. Or go to the file and change the code, because, they are not many categories. 
What do you prefer?

Reply:
I think the yaml file has snake case as format, maybe you can define the serializer after reading the data from yaml files

Reply:
No, I searched information about this and the YAML file can be in Camel Case. 
![Captura desde 2022-12-08 13-34-48](https://user-images.githubusercontent.com/107500715/206509530-0c95ff52-09bc-4f21-995a-e5e876b72013.png)